benchmarks/mpas_ocean.py

```diff
@@ -115,3 +115,37 @@ def teardown(self, resolution, periodic_elements):
 
     def time_dataarray_to_polycollection(self, resolution, periodic_elements):
         self.uxds[data_var].to_polycollection()
+
+
+class ConstructTreeStructures:
+    param_names = ['resolution']
+    params = ['480km', '120km']
+
+    def setup(self, resolution):
+        self.uxds = ux.open_dataset(file_path_dict[resolution][0], file_path_dict[resolution][1])
+
+    def teardown(self, resolution):
+        del self.uxds
+
+    def time_kd_tree(self, resolution):
+        self.uxds.uxgrid.get_kd_tree()
+
+    def time_ball_tree(self, resolution):
+        self.uxds.uxgrid.get_ball_tree()
+
+
+class Remapping:
+    param_names = ['resolution']
+    params = ['480km', '120km']
+
+    def setup(self, resolution):
+        self.uxds = ux.open_dataset(file_path_dict[resolution][0], file_path_dict[resolution][1])
+
+    def teardown(self, resolution):
+        del self.uxds
+
+    def time_nearest_neighbor_remapping(self, resolution):
+        self.uxds["bottomDepth"].remap.nearest_neighbor(self.uxds.uxgrid)
+
+    def time_inverse_distance_weighted_remapping(self, resolution):
+        self.uxds["bottomDepth"].remap.inverse_distance_weighted(self.uxds.uxgrid)
```

Comment:
Looks like the remapping here is from the same grid back onto itself, which may not be the best thing we want to benchmark. Might be a little tricky to handle the `480` to `120` and `120` to `480` benchmarks from the same class.

Maybe split it into two classes:
- `RemapDownsample` (this would be the `120` to `480`)
- `RemapUpsample` (this would be the `480` to `120`) 

The `setup` method can then open both grids.

Reply:
That's a good idea!